packages/docs/content/components/drawer.mdx

```diff
@@ -50,6 +50,44 @@ A header and/or footer can have a "sticky" position if the following properties
   minHeight={300}
 />
 
+### Managing multiple drawers
+
+If you have multiple `<Drawer>`'s on a page, you can utilize the `<DrawerManager>` context provider along with the `useDrawerManager()` hook to manage the state of these drawers so that only one will remain open at a time. The hook provides functions to toggle state of the drawer via click, to close the drawer and to get the open status of the drawer. A minimal example of implementation:
```

Comment:
Maybe we can add a sentence about placing the `DrawerManager` at any level above the components or something like that and then link to general React docs about using providers